browser/ui/webui/settings/brave_wallet_handler.cc

```diff
@@ -53,38 +58,60 @@ void BraveWalletHandler::RegisterMessages() {
 
 void BraveWalletHandler::GetAutoLockMinutes(const base::Value::List& args) {
   CHECK_EQ(args.size(), 1U);
-  PrefService* prefs = Profile::FromWebUI(web_ui())->GetPrefs();
   AllowJavascript();
   ResolveJavascriptCallback(
-      args[0], base::Value(prefs->GetInteger(kBraveWalletAutoLockMinutes)));
+      args[0],
+      base::Value(GetPrefs()->GetInteger(kBraveWalletAutoLockMinutes)));
 }
 
 void BraveWalletHandler::RemoveEthereumChain(const base::Value::List& args) {
   CHECK_EQ(args.size(), 2U);
-  PrefService* prefs = Profile::FromWebUI(web_ui())->GetPrefs();
   AllowJavascript();
+  brave_wallet::RemoveCustomNetwork(GetPrefs(), args[1].GetString());
+  ResolveJavascriptCallback(args[0], base::Value(true));
+}
+
+void BraveWalletHandler::ResetEthereumChain(const base::Value::List& args) {
+  CHECK_EQ(args.size(), 2U);
+  PrefService* prefs = GetPrefs();
+  AllowJavascript();
+  DCHECK(brave_wallet::CustomEthChainExists(prefs, args[1].GetString()));
   brave_wallet::RemoveCustomNetwork(prefs, args[1].GetString());
+  DCHECK(brave_wallet::KnownEthChainExists(args[1].GetString()));
   ResolveJavascriptCallback(args[0], base::Value(true));
 }
 
 void BraveWalletHandler::GetCustomNetworksList(const base::Value::List& args) {
```

Comment:
This function name should probably be updated to GetNetworkLists.

Reply:
https://github.com/brave/brave-core/pull/13644/commits/a14bfe8d849769e50ad78160fe14bd5137bd9aef